pandas/core/base.py

```diff
@@ -877,6 +877,10 @@ def _values(self):
         """ the internal implementation """
         return self.values
 
+    @property
+    def empty(self):
+        return not bool(self.size)
```

Comment:
Do you think it's worth trying to push the implementation in `NDFrame` elsewhere?